packages/graphql/src/schema/create-relationship-fields/create-relationship-fields.ts

```diff
@@ -463,4 +464,9 @@ function createRelationshipFields({
     });
 }
 
+function nodeHasRelationshipWithNestedOperation(node: Node, nestedOperation: RelationshipNestedOperationsOption): boolean {
+ return Boolean(node.relationFields.filter(relationField => relationField.nestedOperations.includes(nestedOperation)).length)
+}
```

Comment:
```suggestion
function nodeHasRelationshipWithNestedOperation(
    node: Node,
    nestedOperation: RelationshipNestedOperationsOption
): boolean {
    return node.relationFields.some((relationField) => relationField.nestedOperations.includes(nestedOperation));
}
```

A slight modification of this to use .some instead. That way we exit the loop early and don't need to do a boolean type conversion.